tests/unit/config/test_package_config.py

```diff
@@ -2318,9 +2318,9 @@ def test_multiple_packages():
     pc.jobs[0].trigger = pc.jobs[0].trigger
     # Accessing package related attributes without 'packages' raises an error
     with pytest.raises(AttributeError, match="ambiguous to get"):
-        pc.jobs[0].enable_net
+        _ = pc.jobs[0].enable_net
```

Comment:
Please don't tell me this affects the evaluation…

Reply:
`ErrorUnknownRequest`

This is to fix https://docs.astral.sh/ruff/rules/useless-expression/

Reply:
Understandable :+1: